config-default.yaml

```diff
@@ -70,4 +70,86 @@ disabledTokens:
       symbol: sUSD-old
       description: This token is disabled for trading and deposit. sUSD will be deprecated and replaced by another token at the end of July.  Go to https://www.synthetix.io for more information
       reason: DEPRECATED
+
+    - address: '0xC011A72400E58ecD99Ee497CF89E3775d4bd732F'
+      name: Synthetix Network Token (deprecated)
+      symbol: SNX-old
+      description: This token is disabled for trading and deposit. SNX will be deprecated and replaced by another token at the end of July.  Go to https://www.synthetix.io for more information
+      reason: DEPRECATED
+      
+    - address: '0x45804880De22913dAFE09f4980848ECE6EcbAf78'
+      name: Paxos Gold
+      symbol: PAXG
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+
+      # aTokens:
+    - address: '0x7D2D3688Df45Ce7C552E19c27e007673da9204B8'
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+      
+    - address: '0xA64BD6C70Cb9051F6A9ba1F163Fdc07E0DfB5F84'
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+
+    - address: '0x71fc860F7D3A592A4a98740e39dB31d25db65ae8'
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+
+    - address: '0x9bA00D6856a4eDF4665BcA2C2309936572473B7E'
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+
+    - address: '0x3a3A65aAb0dd2A17E3F1947bA16138cd37d08c04'
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+
+    - address: '0xfC1E690f61EFd961294b3e1Ce3313fBD8aa4f85d'
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
+
+    - address: '0x9D91BE44C06d373a8a226E1f3b146956083803eB'
+      description: This token is disabled for trading and deposit.
```

Comment:
It feels super repetitive. I would think that adding a flag `disabled` adds already that message `This token is disabled for trading and deposit`. Then, deprecated doesn't need to be modeled. We can just add optionally the `description` for some tokens. These tokens with description would show in the title. 

`"This token is disabled for trading and deposit. " + description`